PencilCase/PencilCase/scripts/Models/Order.ts

```diff
@@ -0,0 +1,54 @@
+﻿import { Product } from './Product';
+
+export class Order {
+    public id: KnockoutObservable<string>;
```

Comment:
Here is a little different from Product model, for each fields, they are defined as observable variables. Because orders need this capacity in the import page, order need to capability to dynamically change quantity and total.

Reply:
And the naming style here is camel, which is different from the way in Product model, I prefer the way in Product, but I didn't want to do such code changes here.